src/server/mod.rs

```diff
@@ -200,8 +201,9 @@ impl LspServer {
         Self {
             client: Arc::new(Mutex::new(client)),
             diagnostics: vec![
-                super::diagnostics::experimental_lint,
                 super::diagnostics::contrib_lint,
+                super::diagnostics::experimental_lint,
+                super::diagnostics::no_influxdb_identifiers,
```

Comment:
For my education only (not requesting change) -- is there a reason for this ordering?